eng/common/build.sh

```diff
@@ -177,6 +177,8 @@ if [[ "$ci" == true ]]; then
 fi
 
 . "$scriptroot/tools.sh"
+InitializeDotNetCli true
+export DOTNET_ROOT=$_InitializeDotNetCli
```

Comment:
undo these changes as they are mirrored by arcade and will be overwritten

Reply:
Hmm, OK, I guess that means that I should make the change in a different place, otherwise I'll need to make it in Arcade.

Reply:
Removed.